temba/triggers/models.py

```diff
@@ -316,6 +318,33 @@ def find_flow_for_inbound_call(cls, contact):
         return trigger.flow
 
     @classmethod
+    def find_trigger_for_ussd_session(cls, contact, starcode):
+        # Determine keyword from starcode
+        keyword = None
+        matched_object = regex.match('(^\*[\d\*]+\#)((?:\d+\#)*)$', starcode)
+        if matched_object:
+            keyword = matched_object.group(1)
+
+            # TODO: use answers for shortcut starcode feature
+            # answers = filter(None, matched_object.group(2).split('#'))
```

Comment:
no this is for the future shortcut feature that will be introduced in a separate PR later. I can just stash it for now!